general-concepts/dependencies/text.xml

```diff
@@ -431,7 +431,9 @@ This can be nested:
 
 <codesample lang="ebuild">
 DEPEND="!build? (
+	&gt;=sys-libs/ncurses-5.2-r2
 	gcj? (
+		&gt;=media-libs/libart_lgpl-2.1
```

Comment:
This example shows its age. It was added in 2008 via [bug 174555](https://bugs.gentoo.org/174555), and was taken straight from gcc-4.2.2.ebuild. Some of the packages listed (e.g. x11-proto/*) no longer exist.

Maybe replacing the example by something more modern would be better?

Reply:
I'm in favour of modernising it, but let's do it as part of a separate PR. I've filed a bug so that I don't forget: https://bugs.gentoo.org/774744.

Reply:
wfm